Project.toml

```diff
@@ -5,9 +5,12 @@ version = "0.4.5"
 
 [deps]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+ChainRulesTestUtils = "cdddcdb0-9152-4a09-a978-84456f9df70a"
 CommonSolve = "38540f10-b2f7-11e9-35d8-d573e4eb0ff2"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 DistributionsAD = "ced4e74d-a319-5a8a-b0ac-84af2272839c"
+FiniteDiff = "6a86dc24-6348-571c-b903-95158fe2bd41"
```

Comment:
I don't think any of these should be added as dependencies, they seem to be only test dependencies?

Reply:
Yes. I think I removed them in the main ones. If I didn't then let me quickly fix those.

Reply:
@devmotion I removed those but forget to push :( now it's updated. Thanks